helm-chart/renku-core/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "renku-core.fullname" . }}-test-connection"
+  labels:
+{{ include "renku-core.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args:  ['{{ include "renku-core.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Does this actually work? Maybe query the `/health` endpoint to test the connection?

Reply:
oh I think this was automatically generated  by `helm` - I will remove it